README.md

```diff
@@ -68,7 +68,7 @@ ipython
 import deepchem as dc
 ```
 
-### Installing from source in a conda environment
+### Installing from the source in a conda environment

```

Comment:
'Installing from source' is common terminology and I feel it should be left that way

Reply:
I created a new pull request considering the suggestions